cuml/linear_regression_demo.ipynb

```diff
@@ -105,19 +110,46 @@
    "source": [
     "## Scikit-learn Model\n",
     "\n",
+    "Copy dataset from GPU memory to host memory.\n",
+    "This is done to later compare CPU and GPU results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X_train = X_cudf.to_pandas()\n",
```

Comment:
Why do we do this if we already define and split the dataset `X, y` into `X_train, X_test, y_train, y_test`

Reply:
Not needed anymore.

Fixed in local.

P.S.: I had it because, in order to compare both results, I wanted to make sure that the dataset in the Host and in the GPU were the same. With the new approach, it is not needed anymore.